spec/sanity/events.js

```diff
@@ -1,38 +1,199 @@
-//'use strict';
-//
-//var logger = require('log4js').getLogger('example_spec');
-//var components = require('../src/components');
-//
-//describe('Events page:', function () {
-//
-//    beforeEach(function (done) {
-//        components.ui.events.IndexPage.beforeEach(done);
-//    });
-//
-//    it('should navigate to logs & events', function (done) {
-//        expect(components.ui.events.IndexPage.getNumOfEvents()).not.toBe(0);
-//
-//        browser.sleep(1000).then(function () { done(); });
-//    });
-//
-//    it('should list events for blueprint', function(done) {
-//        components.ui.events.IndexPage.uncheckAllBlueprints();
-//        components.ui.events.IndexPage.selectBlueprint('nodecellar1');
-//        components.ui.events.IndexPage.selectTimeframe('5 Days');
-//        components.ui.events.IndexPage.show();
-//
-//        browser.sleep(5000).then(function(){ done(); });
-//    });
-//
-//    it('should list events for blueprint and deployment', function(done) {
-//        components.ui.events.IndexPage.uncheckAllBlueprints();
-//        components.ui.events.IndexPage.selectBlueprint('nodecellar1');
-//        components.ui.events.IndexPage.uncheckAllDeployments();
-//        components.ui.events.IndexPage.selectDeployment('deployment1');
-//        components.ui.events.IndexPage.selectTimeframe('5 Days');
-//        components.ui.events.IndexPage.show();
-//
-//        browser.sleep(5000).then(function(){ done(); });
-//    });
-//
-//});
\ No newline at end of file
+'use strict';
+var logger = require('log4js').getLogger('blueprints_spec');
+var components = require('../../src/components/index');
+var events = components.ui.events.page;
+
+describe('logs & events page', function() {
+
+    beforeEach(function(){
+        events.route();
+    });
+    
+    it('should route to events page', function(){
+        expect(browser.getCurrentUrl()).toContain('/logs');
+    });
+
+    //TODO: make it after using api
+    //it('should sort by timestamp desc by default', function(){
+
+    //});
+
+    it('should filter blueprints', function(){
+        //getting number of events on startup
+        var noFiltersEventsCount = events.mainTable.countRows();
+        //Choosing a blueprint with no events
+        events.filters.blueprints.select('nodecellar_undeployed');
```

Comment:
too many hard coded values. 

lets at least define a constants object with all the string so if we ever need to change that, it will be easy.